test/suite/test_txn08.py

```diff
@@ -72,6 +72,39 @@ def test_printlog_unicode(self):
             '\\u0001\\u0002abcd\\u0003\\u0004')
         self.check_file_contains('printlog-hex.out',
             '0102616263640304')
+        # Check the printlog start LSN and stop LSN feature.
+        self.runWt(['printlog', '-l 2,128'], outfilename='printlog-range01.out')
+        self.check_file_contains('printlog-range01.out',
+            '"lsn" : [2,128],')
+        self.check_file_contains('printlog-range01.out',
+            '"lsn" : [2,256],')
+        self.check_file_not_contains('printlog-range01.out',
+            '"lsn" : [1,128],')
+        self.runWt(['printlog', '-l 2,128,3,128'], outfilename='printlog-range02.out')
+        self.check_file_contains('printlog-range02.out',
+            '"lsn" : [2,128],')
+        self.check_file_not_contains('printlog-range02.out',
+            '"lsn" : [1,128],')
+        self.check_file_not_contains('printlog-range02.out',
+            '"lsn" : [3,256],')
+        # Test for invalid lsn, return WT_NOTFOUND
+        self.runWt(['printlog', '-l 2,100'], outfilename='printlog-range03.out', errfilename='printlog-range03.err', failure=True)
```

Comment:
The header of the log file is in the first 128 bytes, please check a value larger.